settings.yaml

```
@@ -11,6 +11,11 @@ enabled_features:
     _HOST_dev: true
     _HOST_test: true
     _HOST_prod: true
+  cloud_platform_assume_role:
+    _DEFAULT: false
+    _HOST_dev: false
```

```suggestion
    _HOST_dev: true
```
Should we enable it in dev for testing?